### PR TITLE
Test suite, more package names

### PR DESCRIPTION
**Pull request by dvc94ch**

### CODE REVIEW

File: js/lib/electro_grammar_listener.js [hunk 1]
```diff
@@ -1,47 +1,145 @@
 const {ElectroGrammarListener} = require('./ElectroGrammarListener');
+const _ = require('underscore');
+
+function handle_option(ctx, options) {
+  const key = _.find(Object.keys(options), (key) => {
```
Comment (kasbah) [inline js/lib/electro_grammar_listener.js]: Do you need underscore? You can do `Object.keys(options).find(key => {...`

**Comment by kasbah** [comment]
It was easier to just merge these? It does make it harder to check if previously reviewed stuff has been changed. 121038b should not be included in this. Also, you left behind 3 commits I made that enabled CI testing.

**Comment by kasbah** [comment]
Ok, running the code formatters should make Travis happy and then it can be merged.

File: .gitignore [hunk 2]
```diff
@@ -5,4 +5,5 @@ test-lib/
 
 java/
 js/**/ElectroGrammar*
-python3/**/ElectroGrammar*
\ No newline at end of file
+python3/**/ElectroGrammar*
+db/*.json
```
Comment (kasbah) [inline .gitignore]: This is another minor thing but could you configure your editor to add new lines at the end of files?

File: test/test_units.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      1e-6: ['1us'],
+      1e-9: ['1ns'],
+      1e-12: ['1ps']
+    };
+    lexer_tests(config, cases, 'time', Number);
+  });
+
+  describe('temperature', () => {
+    const cases = {
+      1: ['1°C', '1C']
```
[the remark this answers is not in the document]
Comment (dvc94ch) [reply test/test_units.js]: Yes, you're right. I suggest we revisit this then dough.

I'll be working on `key (=|>=|<=|<|>) value` next, but I'm taking a break to hack on qeda/netlistsvg and migrating the pycircuit component library to qeda. I think the work on qeda/netlistsvg will benefit replicad and electrogrammar too.

FYI: I want to make the strict parser more strict when it comes to units. 1V is a voltage, 1v, 1volt, 1Volt should not be voltages in strict mode.

File: .gitignore (hunk 2 above)
Comment (dvc94ch) [reply .gitignore]: I do have it configured like that. I think I may have used `echo 'db/*.json'>>.gitignore`

**Pull request merged**